impress-org/givewp
# Add support for donation refunding

Resolves #4984 

## Description

This PR hooks into the event of a donation being marked as refunded and sends the request to PayPal to have the payment refunded on their end. I feel like I should say more, but that's pretty much all it does! 😁 

## Affects

When a donation is marked as refunded the payment is refunded.

## Pre-review Checklist




## Testing Instructions

1. Make a donation
2. Go to the donation in the admin
3. Set the donation status to "refunded" and update the donation
4. Log into the PayPal sandbox account and check if the payment has been refunded
